kratos/utilities/specifications_utilities.cpp

```diff
@@ -442,15 +552,15 @@ std::string DetermineFramework(ModelPart& rModelPart)
                 framework = r_framework;
             } else {
                 CompareElementsAndConditionsUtility::GetRegisteredName(*it_cond_begin, condition_name);
-                KRATOS_WARNING("SpecificationsUtilities") << "Framework:" << r_framework << " is used by the condition: " << condition_name << ". But also the framework: " << framework << " is used" << std::endl; 
+                KRATOS_WARNING("SpecificationsUtilities") << "Framework:" << r_framework << " is used by the condition: " << condition_name << ". But also the framework: " << framework << " is used" << std::endl;
```

Comment:
Not sure if this has to be a warning. It is perfectly possible to mix Eulerian elements with ALE ones.

Reply:
And Lagrangian with Eulerian, or ALE, I agree. This is a draft, I am doing what the @KratosMultiphysics/technical-committee  tells me, and the lack of details makes me to take some ambiguous decisions

Reply:
just to say that we are also not sure what should be there...

Reply:
> just to say that we are also not sure what should be there...

![imagen](https://user-images.githubusercontent.com/19991680/78149957-ea871a00-7436-11ea-955c-1740e8e57e48.png)